lib/FluxibleContext.js

```diff
@@ -82,8 +82,9 @@ FluxContext.prototype.plug = function (plugin) {
 FluxContext.prototype.executeAction = function executeAction(action, payload, done) {
     var self = this;
     payload = payload || {};
+    var displayName = action.name || action.displayName;
```

Comment:
I wonder if we should switch this precedence: `var displayName = action.displayName || action.name;`